### ISSUE TITLE
Stage: LUP user does not have the ability to save any progress/changes to the disposition record prior to submission 

### ISSUE THREAD

**Issue by ksinghprod** [NYCPlanning/labs-zap-search#1053]
Browser: Chrome
Project: any hearing or recommendation letter 

Expected Result: LUP should have the ability to save any changes to the disposition record at any time prior to submission 
Actual Result:  LUP user does not have the ability to save any progress/changes to the disposition record prior to submission 

As of now the LUP user has to submit the hearing&recommendation once they begin, they do no have to ability to save and come back to it later. 

![image](https://user-images.githubusercontent.com/57362229/70256097-e03d6380-1755-11ea-9043-ad8c7456dc74.png)

**Comment by allthesignals** [comment]
Good catch, but we didn't design for this. Let's save this in an issue if/when it comes up post-release. Enhancement.
